### PR TITLE
chore(ci): cache docker images in packer

### PR DESCRIPTION
**Pull request by mdelapenya**
## What does this PR do?
It adds picklesdoc and golang docker images in Packer

## Why is it important?
It will reduce build time

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the Unit tests for the CLI, and they are passing locally
- [ ] I have run the End-2-End tests for the suite I'm working on, and they are passing locally
- [ ] I have noticed new Go dependencies (run `make notice` in the proper directory)

## Author's Checklist

- [x] Checked the new script returns the proper GO_VERSION

## Follow-ups
Identify new images and push updates in the script.

Question: is it possible to run the clean-docker.sh script at packer time? If so, would it be too much size to cache all those images?

### CODE REVIEW

File: .ci/packer_cache.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+_sdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# shellcheck disable=SC1091
+source /usr/local/bin/bash_standard_lib.sh
+
+readonly GO_VERSION=$(cat ${_sdir}/../.go-version)
```
Comment (v1v) [inline .ci/packer_cache.sh:8]: The script runs in the context of the workspace, therefore if .go-version is in the top level folder, you could simplify the _sdir 

```
 readonly GO_VERSION=$(cat .go-version)
```

**Review by v1v — CHANGES_REQUESTED** [review]

File: .ci/packer_cache.sh (hunk 1 above)
Comment (mdelapenya) [inline .ci/packer_cache.sh:8]:
```suggestion
# shellcheck disable=SC1091
source /usr/local/bin/bash_standard_lib.sh

readonly GO_VERSION=$(cat .go-version)
```

**Review by cachedout — APPROVED** [review]

**Review by kuisathaverat — APPROVED** [review]

**Comment by mdelapenya** [comment]
@kuisathaverat are we usually backporting packer cache scripts to the maintenance branches?

**Comment by kuisathaverat** [comment]
> @kuisathaverat are we usually backporting packer cache scripts to the maintenance branches?

no, the script is only grabbed from the master so you do not need to backport the change

**Pull request merged**